README.md

```diff
@@ -231,10 +230,9 @@ components: {
   },
 ```
 
-When the `devMode` is enabled. You will see information boxes at the address the server is running on. 
-There is described which component you have to implement and which information you will be getting. 
-For more information you will soon be able to visit our getting started guide of our [FSXA-PWA](https://github.com/e-Spirit/fsxa-pwa) project for more information.
+Or you can follow the naming convention and create the appropriate folder structure.
 
+If you specify the `layout`, `section` and `richtest` directory you can place and name the folders anywhere you like as long as the actual path is matching the path that is written in the component object.
```

Comment:
A comma is missing between `section` and 'and'.

Reply:
My research has yielded that a comma before an 'and' in an enumeration is optional

Reply:
Yes but to us that means learning two conventions because in German using a comma before an and is a no no ;)